documents/knowledge/elements.yaml

```diff
@@ -3695,6 +3697,17 @@
     - "[8] : 5"
     - "[9] : 3"
 
+- element: "∆Ė"
+  name: N Digits of E / Sympy Evaluate
```

Comment:
Specify that `E` is Euler's constant? It took me a while to work out the it wasn't an argument. Will also help with searching

Reply:
[Euler's constant](https://en.wikipedia.org/wiki/Euler's_constant) is another number entirely. It's kinda confusing, and I really like `e` better than Euler's number because of that.